docs/self-managed/setup/deploy/amazon/amazon-eks/eks-helm.md

```diff
@@ -25,7 +25,7 @@ Multi-tenancy is disabled by default and is not covered further in this guide. I
 
 ### Architecture
 
-Note the [existing architecture](/self-managed/platform-architecture/overview.md#architecture) extended by deploying a Network Load Balancer with TLS termination within the [ingress](https://kubernetes.github.io/ingress-nginx/user-guide/tls/) below.
+Note the [existing architecture](../../../../about-self-managed.md#architecture) extended by deploying a Network Load Balancer with TLS termination within the [ingress](https://kubernetes.github.io/ingress-nginx/user-guide/tls/) below.
```

Comment:
I got into a habit of using the docs area instead of trying to walk the file structure with `../`'s. Can you bring this to a team meeting so we can discuss it? No action required here.